### PR TITLE
feat: post code snippets query

### PR DESCRIPTION
**Pull request by capJavert**

### CODE REVIEW

File: __tests__/fixture/post.ts [hunk 1]
```diff
@@ -397,3 +399,31 @@ export const contentUpdatedPost = {
...
+  postId,
+}: {
+  postId: string;
+}): DeepPartial<PostCodeSnippet>[] => [
+  {
+    postId,
+    contentHash: '865cd55417af5a27ab17ae1fff7510c7acc4f250',
+    order: 0,
+    language: PostCodeSnippetLanguage.Plain,
+    content: 'console.log("Hello World")',
```
Comment (rebelchris) [inline __tests__/fixture/post.ts:413]: Not related to this specific PR, but we are sanitizing what we crawl right before storing?

**Review by rebelchris — APPROVED** [review]

File: __tests__/fixture/post.ts (hunk 1 above)
Comment (capJavert) [reply __tests__/fixture/post.ts:413]: We are not doing any processing on snippets on API side. What would you sanitize?

Comment (rebelchris) [reply __tests__/fixture/post.ts:413]: More wondering if someone could have malicious snippets that on insert in pg break anything?

Comment (capJavert) [reply __tests__/fixture/post.ts:413]: we are using typeorm and INSERT query it does is parameterized so it should be good

```sql
INSERT INTO "public"."post_code_snippet"("postId", "contentHash", "order", "language", "createdAt", "content") VALUES ($1, $2, $3, DEFAULT, DEFAULT, $4), ($5, $6, $7, DEFAULT, DEFAULT, $8)
```

Comment (rebelchris) [reply __tests__/fixture/post.ts:413]: ah perfect :)

**Pull request merged**